### PR TITLE
Adding new templates

### PR DESCRIPTION
**Pull request by wallentx**
This is to aid in release note generation.

### CODE REVIEW

File: .github/workflows/require-labels.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    outputs:
+      status: ${{ steps.check-labels.outputs.status }}
+    steps:
+      - id: check-labels
+        uses: mheap/github-action-required-labels@v2
+        with:
+          mode: exactly
+          count: 1
+          labels: "Added, Changed, Fixed"
+          exit_type: success
```
Comment (cmmarslender) [inline .github/workflows/require-labels.yml:21]: We'd need to allow the other non-changelog labels too right? (CI, etc)

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: .github/workflows/require-labels.yml (hunk 1 above)
Comment (wallentx) [reply .github/workflows/require-labels.yml:21]: My thinking was that even release-note-excluded changes can fit into one of these 3 categories.
My 🧠
- Maintain baseline requirements: always
- Add exceptions as an additional layer: can be tweaked later on without disrupting our baseline truth

Comment (wallentx) [reply .github/workflows/require-labels.yml:21]: Que 👍 or 👎

Comment (cmmarslender) [reply .github/workflows/require-labels.yml:21]: Yeah I guess that makes sense. We can always change it later if we need.

**Review by cmmarslender — DISMISSED** [review]

**Comment by wallentx** [comment]
Some other templates for Pull Requests if anyone has feelings on fine-tuning `.github/PULL_REQUEST_TEMPLATE.md`

https://github.com/axolo-co/pull_request_template
https://github.com/devspace/awesome-github-templates

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+* **What is the new behavior (if this is a feature change)?**
+
+
+
+* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
+
+
+
+* **Other information**:
```
Comment (paninaro) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: Personally I would like to see this swapped with
```suggestion
* **Testing notes (is this code covered by tests, or equivalent manual testing?)**```

**Review by paninaro — DISMISSED** [review]

**Review by paninaro — DISMISSED** [review]

**Review by paninaro — APPROVED** [review]

**Pull request merged**